### PR TITLE
Post store rate

### PR DESCRIPTION
**Pull request by orangeOAO**
POST /store/{storeID}/rate

### CODE REVIEW

File: backend/store/delivery/store_handler.go [hunk 1]
```diff
@@ -96,3 +98,19 @@ func (s *StoreHandler) GetSelling(c *gin.Context) {
...
+	if err != nil {
+		return
+	}
+	var rate swagger.RateInfo
+	if err := c.BindJSON(&rate); err != nil {
+		logrus.Error(err)
+		c.Status(400)
+		return
+	}
+	s.StoreUsecase.CalculateRate(c, storeID, rate)
```
Comment (wj461) [inline backend/store/delivery/store_handler.go]:
```suggestion
	err := s.StoreUsecase.CalculateRate(c, storeID, rate)
	if err := nil {
	    logrus.Error(err)
		c.Status(500)
		return
	}
	
	c.Status(200)
```

Comment (orangeOAO) [reply backend/store/delivery/store_handler.go]: fix

**Comment by wj461** [comment]
You can look readme format part to auto format

**Comment by wj461** [comment]
I think you can merge upsteam

**Review by nonorze-VOXX — CHANGES_REQUESTED** [review]
擋一下評價分數的範圍

**Review by wj461 — APPROVED** [review]
looks good

**Review by nonorze-VOXX — APPROVED** [review]
LGtm

**Pull request merged**